packages/mdc-top-app-bar/mdc-top-app-bar.scss

```diff
@@ -145,5 +159,24 @@
   }
 }
 
+// Specific styles for prominent and dense styled top app bar
+// stylelint-disable plugin/selector-bem-pattern
+.mdc-top-app-bar--dense.mdc-top-app-bar--prominent {
+  .mdc-top-app-bar__row {
+    height: $mdc-top-app-bar-dense-row-height * 2;
+  }
+
+  .mdc-top-app-bar__section {
+    padding: 0 $mdc-top-app-bar-section-horizontal-padding;
+  }
+
+  .mdc-top-app-bar__title {
+    @include mdc-rtl-reflexive-box(padding, left, $mdc-top-app-bar-title-left-padding);
+
+    padding-bottom: $mdc-top-app-bar-prominent-dense-title-bottom-padding;
```

Comment:
This is also overriding mobile-specific title bottom padding for prominent (not dense). Is that intentional?

(TBH I find it hard to follow the plethora of different title paddings but I get the impression it's to counteract outer padding...)

Reply:
It's intentional. The `--dense` style is desktop only, so any styles it applies should override any other styles from the mobile media queries.